sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -996,6 +996,31 @@ def test_negative_fractions():
     expr = Mul(Rational(2, 3), Rational(5, 7), evaluate=False)
     assert pretty(expr) == "2/3*5/7"
     assert upretty(expr) == "2/3⋅5/7"
+    expr = Mul(x + y, Rational(1, 2), evaluate=False)
+    assert pretty(expr) == "x + y\n-----\n  2  "
+    assert upretty(expr) == "x + y\n─────\n  2  "
+    expr = Mul(Rational(1, 2), x + y, evaluate=False)
+    assert pretty(expr) == "x + y\n-----\n  2  "
+    assert upretty(expr) == "x + y\n─────\n  2  "
+    expr = Mul(S.One, x + y, evaluate=False)
+    assert pretty(expr) == "1*(x + y)"
+    assert upretty(expr) == "1⋅(x + y)"
+    expr = Mul(x - y, S.One, evaluate=False)
+    assert pretty(expr) == "x - y"
+    assert upretty(expr) == "x - y"
+    expr = Mul(Rational(1, 2), x - y, S.One, x + y, evaluate=False)
+    assert pretty(expr) == "1/2*(x - y)*1*(x + y)"
+    assert upretty(expr) == "1/2⋅(x - y)⋅1⋅(x + y)"
+    expr = Mul(x + y, Rational(3, 4), S.One, y - z, evaluate=False)
+    assert pretty(expr) == "3*(x + y)*(y - z)\n-----------------\n        4        "
```

Comment:
I don't see how https://github.com/sympy/sympy/pull/21819#issuecomment-894163417 is true.

Reply:
@namannimmo10  On this branch , In my local setup
```
>>> import sympy
>>> from sympy import *
>>> x,y,z = symbols('x y z')
>>> Mul(x + y, Rational(3, 4), S.One, y - z, evaluate=False)
```
outputs `(x + y)⋅(3/4)⋅1⋅(y - z)`

Reply:
@namannimmo10 This is fixed now ,
`>>> pretty(Mul(x + y, Rational(3, 4), S.One, y - z, evaluate=False))`
outputs `(x + y)*1*3/4*5/6`  and
`>>> upretty(Mul(x + y, Rational(3, 4), S.One, y - z, evaluate=False))`
outputs `(x + y)⋅1⋅3/4⋅5/6`